troubleshooting/missing-image-asset-causes-ios-publish-to-fail.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Error Publishing App to the iOS AppStore
```

Comment:
This is a bit too general as you might get many types of errors. Unless you're planning on including all errors that our users have experienced in this article, consider making it more specific.This can be easily achieved by citing the error or a non-ambiguous part of it.

Also, the correct trademark name is App Store. You can go with either "the App Store" or "the Apple App Store"